connect/json/src/main/java/org/apache/kafka/connect/json/JsonConverter.java

```diff
@@ -304,7 +304,7 @@ public void configure(Map<String, ?> configs, boolean isKey) {
 
         Object cacheSizeVal = configs.get(SCHEMAS_CACHE_SIZE_CONFIG);
         if (cacheSizeVal != null)
-            cacheSize = (int) cacheSizeVal;
+            cacheSize = Integer.parseInt((String) cacheSizeVal);
```

Comment:
Actually, they're all casted explicitly to Map<String,String>

https://github.com/christian-posta/kafka/blob/c9485b78a6e43747daf1314ae9532839fb7bc810/connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectDistributed.java#L61-61

https://github.com/christian-posta/kafka/blob/18f643b82b05904516295e2734af0d8a9fb053ea/connect/runtime/src/main/java/org/apache/kafka/connect/cli/ConnectStandalone.java#L66-66

And both StandaloneConfig and DistributedConfig accept only Map<String,String>

https://github.com/christian-posta/kafka/blob/079c88178dff4b3a4c9de55629e7d15b60e5f562/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/DistributedConfig.java#L197-197

https://github.com/christian-posta/kafka/blob/079c88178dff4b3a4c9de55629e7d15b60e5f562/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java#L42-42

I tried change them all to Map<String,Object> but that caused other problems.

Thoughts?